config/authorities/name_type.yml

```diff
@@ -2,6 +2,6 @@ terms:
 - id: Personal
   term: Personal
   active: true
-- id: Organisational
+- id: Organizational
```

Comment:
All of this code is so this is possible and doesn't break the form toggling and DOI